### ISSUE TITLE
NamedParameter.Type is not being parsed from queryParameters defined using Traits

### ISSUE THREAD

**Issue by fstaoe** [Jumpscale/go-raml#332]
**Issue:** 
Using a trait as defined on the example below, the `type` inside the `queryParameters` is not being unmarshaled and the resource method which inherits those query parameters is getting all the properties such as `name`, `description`, `default`, `minimum`, etc but the `type` is always an empty string. 

Example:
```
  paged:
    usage: Applies the limit and the offset parameters to pagination purposes
    queryParameters:
      offset:
        description: Description for offset property
        type: integer
        default: 0
        minimum: 0
        required: false
      limit:
        description: Description for limit property
        type: integer
        default: 30
        minimum: 0
        required: false
```

**Initial Request**
Could you please clarify if it is a bug on the parser or a wrong definition?

**Comment by iwanbk** [comment]
Hi, could you reformat your raml example?

**Comment by iwanbk** [comment]
It is a bug, fixes on #333
